internal/deploy/kubectl/websphereliberty-app-operator.yaml

```diff
@@ -0,0 +1,357 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/instance: websphere-liberty-operator
+    app.kubernetes.io/managed-by: wlm
```

Comment:
this should remain as `olm` - referring to Operator Lifecycle Manager. Same for all instances below

Reply:
Fixed in latest commit